elixir/apps/web/lib/web/live/settings/api_clients/beta.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Web.Settings.ApiClients.Beta do
+  use Web, :live_view
+
+  def mount(_params, _session, socket) do
+    socket =
+      socket
+      |> assign(:page_title, "API Clients")
+
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~H"""
+    <.breadcrumbs account={@account}>
+      <.breadcrumb path={~p"/#{@account}/settings/api_clients"}>API Clients</.breadcrumb>
+      <.breadcrumb path={~p"/#{@account}/settings/api_clients/beta"}>Beta</.breadcrumb>
+    </.breadcrumbs>
+
+    <.section>
+      <:title><%= @page_title %></:title>
+      <:help>
+        API Clients are used to manage Firezone configuration through a REST API. See the
+        <a class={link_style()} href="https://firezone.dev/kb/reference/rest-api">REST API docs</a>
+        for more info.
+      </:help>
+      <:content>
+        <div class="max-w-2xl px-4 py-2 mx-auto">
+          <div class="text-lg text-center">
+            The Firezone REST API is currently in closed beta.  You may request access by emailing
+            <.link
+              class={link_style()}
+              href={
+                mailto_support(@account, @subject, "REST API Closed Beta Request: #{@account.name}")
+              }
+            >
+              support@firezone.dev
+            </.link>
+            with the following message template.
+          </div>
+          <div class="my-4">
+            Subject: <.code_block
+              id="msg_subject"
+              class="w-full text-xs whitespace-pre-line"
+              phx-no-format
+              phx-update="ignore"
+            >REST API Closed Beta Request: <%= "#{@account.name}" %></.code_block>
+          </div>
+
+          <div class="my-4">
+            Body: <.code_block
+              id="msg_body"
+              class="w-full text-xs whitespace-pre-line"
+              phx-no-format
+              phx-update="ignore"
+            ><%= email_body(@account, @subject) %></.code_block>
+          </div>
+        </div>
```

Comment:
I think this can be simplified / improved in a couple ways:

- We can encode everything here in the `mailto` link itself. See the `mailto_support/3` function in core_components.ex for example:
```
  def mailto_support(account, subject, email_subject) do
    body =
      """


      ---
      Please do not remove this part of the email.
      Account Name: #{account.name}
      Account Slug: #{account.slug}
      Account ID: #{account.id}
      Actor ID: #{subject.actor.id}
      """

    "mailto:support@firezone.dev?subject=#{URI.encode_www_form(email_subject)}&body=#{URI.encode_www_form(body)}"
  end
```
- It might look a tad better if we style this similar to a flash info. What do you think?
```
<div id="flash" class="p-4 text-sm flash-info text-blue-800 bg-blue-50 mb-4 rounded" role="alert">
  <p class="flex items-center gap-1.5 text-sm font-semibold leading-6">
    <span class="hero-wrench-screwdriver h-4 w-4"></span>
    
   REST API Beta
  </p>The REST API is currently in closed beta. <a href="#" class="text-accent-900 underline">Click here</a> to request access.
</div>
```
<img width="720" alt="Screenshot 2024-07-24 at 10 10 57 PM" src="https://github.com/user-attachments/assets/6c59f1f8-5d3e-4171-943f-3df07f09bb68">

Reply:
I'm not wholly opposed to this suggestion, but I am curious about gathering a little more data.  

As I started working on this, I had planned to do something very similar to what you've suggested here, however, I realized that I couldn't even test it out, because I never use `mailto` links since I don't have any mail client setup on my machine.  When I would click on it, it would open up Apple Mail and ask me to setup my mail provider, etc...  I would just click cancel on all of that, but that left me back on the web page back where I started and with no info on what email address I needed to send the request to and also no idea of what message I should send.  That's when I changed my approach slightly and made it more explicit.  I did use the `mailto_support` function on line 33, so people have the option to use that, but they also know explicitly what needs to be sent if they don't use `mailto` links.

The thing I'm curious about is, am I an extreme outlier or is there a good chance other people don't use `mailto` links directly?

Reply:
For this feature actually, I would suggest just automatically delivering us a simple text-based email to `support@` when they click a `Request access` button and show some success message.

Support is a bit different because we need them to describe to us what the issue is, but it would also be better there to open a modal for their feedback and just email it to us as well.

Reply:
Agree with all of that.  I had considered doing this, but talked myself out of it because I was thinking about the scenario of people clicking the button constantly, but that may not really be that big of a concern.  Are you ok if they can click the 'Request Access' button more than once?

Reply:
Yeah that's fine.

Reply:
👍 Cool, I'll start working on this and I'll use your 'flash info' suggestion, that does look nice.